tools/meta_parent_policy_compiler/azure_meta_parent.pt.template

```diff
@@ -749,7 +758,11 @@ script "js_take_in_parameters", type: "javascript" do
     console.log("Should keep "+account_id+"="+keep);
     if (!keep) {
       if (grid_and_cwf.to_delete.length < max_actions) {
-        grid_and_cwf.to_delete.push({"id":ds_format_existing_policies.formatted[account_id].applied_policy_id, "name": ds_format_existing_policies.formatted[account_id].applied_policy_name})
+        grid_and_cwf.to_delete.push({
+          "accountID": account_id,
+          "id": ds_format_existing_policies.formatted[account_id].applied_policy_id,
+          "name": ds_format_existing_policies.formatted[account_id].applied_policy_name
+        })
```

Comment:
```suggestion
        grid_and_cwf.to_delete.push({
          "accountID": account_id,
          "id": ds_format_existing_policies.formatted[account_id].applied_policy_id,
          "name": ds_format_existing_policies.formatted[account_id].applied_policy_name
        })
        
```
To make sure Azure Meta Parent template is consistent with the AWS and Google ones